Formula/libxml2.rb

```diff
@@ -70,10 +71,14 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
                           "--with-history",
+                          "--with-icu",
```

Comment:
Revision bump?

Reply:
Should this be handled as a separate PR?  I'm fine with revision bumping it here but if we enable dependent testing it will be a much longer run for Linux.

Reply:
Yes, though I'm not entirely sure why we actually need this since we don't for Qt6?

Reply:
I’m not entirely sure - QtWebEngine in Qt6 can use the brewed`libxml2` On Linux with no issues.  Regardless as I’ll explain in that PR, Arch Linux and Debian at least build `libxml2` with ICU support so this is not an unusual feature to include for it.